### PR TITLE
[IT] Update `Mascots`

### PR DESCRIPTION
**Pull request by Nakkeraru**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Mascots/it.md [hunk 1]
```diff
@@ -1,290 +1,101 @@
...
+Disegnata da [\[ Glitch \]](https://osu.ppy.sh/users/3781400), Alisa era una delle partecipanti del design contest delle mascotte di osu!taiko. Gioca ad osu!taiko fin da bambina. Adora suonare canzoni per gli altri, ama la musica, e quando non sta dormendo o mangiando, gioca ad osu!taiko oppure a qualche videogioco retró per divertimento!
 
-Maschio
+### Chirou
 
-#### Data di inizio
+![Chirou](img/Chirou.png "Chirou")
 
-May 2008
+Disegnata da [pyun](https://osu.ppy.sh/users/981534), Chirou era una delle partecipanti del design contest delle mascotte di osu!taiko. É molto severa e esigente, una perfezionista che non ama commettere errori soprattutto mentre gioca a osu!taiko. Ma al di lá del suo aspetto severo, se riesci a raggiungere il suo lato dolce, puó essere gentile e premurosa. Ha 14 anni, nata il 25 Ottobre, gruppo sanguigno AB. É alta 150cm e pesa 45Kg. I suoi hobby includono esercitarsi a suonare la batteria, stare al caldo con la sua coperta e collezionare gemme o rocce. Chirou è apparsa in diverse fanart create dagli utenti.
```
Comment (Sclafus) [inline wiki/Mascots/it.md:89]:
```suggestion
Disegnata da [pyun](https://osu.ppy.sh/users/981534), Chirou era una delle partecipanti del design contest delle mascotte di osu!taiko. È molto severa e esigente, una perfezionista che non ama commettere errori soprattutto mentre gioca ad osu!taiko. Ma al di là del suo aspetto severo, se riesci a raggiungere il suo lato dolce, può essere gentile e premurosa. Ha 14 anni, nata il 25 Ottobre, gruppo sanguigno AB. È alta 150cm e pesa 45Kg. I suoi hobby includono esercitarsi a suonare la batteria, stare al caldo con la sua coperta e collezionare gemme o rocce. Chirou è apparsa in diverse fanart create dagli utenti.
```

File: wiki/Mascots/it.md [hunk 2]
```diff
@@ -1,290 +1,101 @@
...
-#### Creator
+### ![icona Osu!](/wiki/shared/mode/osu.png) pippi
 
-- Sarumaru (Concept e arte originale)
-- Daru (Arte corrente)
-- crystalsuicune (pippidon)
+![pippi](img/pippi.png "pippi")
 
-#### Genere
+pippi, stilizzata con una "p" minuscola, è la mascotte della modalità di gioco osu! dal 2008-07. È anche conosciuta come pippidon in osu!taiko e appare in [Yandere Simulator](https://yanderesimulator.com) come un NPC. La concept art iniziale è stata creata da [Sarumaru](https://osu.ppy.sh/users/9427), la grafica é stata creata da [crystalsuicune](https://osu.ppy.sh/users/9974), e il disegno attuale é stato progetto da [Daru](https://osu.ppy.sh/users/32480).
```
Comment (Sclafus) [inline wiki/Mascots/it.md:13]:
```suggestion
pippi, stilizzata con una "p" minuscola, è la mascotte della modalità di gioco osu! dal 2008-07. È anche conosciuta come pippidon in osu!taiko e appare in [Yandere Simulator](https://yanderesimulator.com) come un NPC. La concept art iniziale è stata creata da [Sarumaru](https://osu.ppy.sh/users/9427), la grafica è stata creata da [crystalsuicune](https://osu.ppy.sh/users/9974), e il disegno attuale è stato progetto da [Daru](https://osu.ppy.sh/users/32480).
```

File: wiki/Mascots/it.md [hunk 3]
```diff
@@ -1,290 +1,101 @@
...
+*Per ulteriori informazioni, consulta: [Presentiamo Mani e Mari, le nuove mascotte di osu!mania.](https://osu.ppy.sh/home/news/2020-09-17-introducing-mani-mari-osumania)*
 
-#### Aspetto
+![Mani & Mari](https://assets.ppy.sh/media/mari-mani/wiki-key-condensed.png "Mani & Mari")
 
-Mania
+Disegnati da [xiemon](https://osu.ppy.sh/users/5203667) dopo essere stati selezionati nella [Most Manic Art Contest](https://osu.ppy.sh/community/contests/80) e annunciati come le mascotte ufficiali di osu!mania, Mani e Mari sono gemelli che non riescono a sopportarsi a vicenda, ma sono più simili di quanto vorrebbero ammettere.
 
-#### Creator
+Mani è un po' un ribelle, sempre alla ricerca di nuovi stili e cose da esplorare, mentre sua sorella Mari (precedentemente conosciuta come Maria) è una classica perfezionista, rigida con una predilezione per i riflettori. Vanno d'accordo come l'olio e l'acqua.
```
Comment (Sclafus) [inline wiki/Mascots/it.md:31]:
```suggestion
Mani è un po' un ribelle, sempre alla ricerca di nuovi stili e cose da esplorare, mentre sua sorella Mari (precedentemente conosciuta come Maria) è una classica perfezionista, rigida e le piace stare al centro dell'attenzione. Vanno d'accordo come l'olio e l'acqua.
```

File: wiki/Mascots/it.md [hunk 4]
```diff
@@ -1,290 +1,101 @@
...
+### ![icona osu!taiko ](/wiki/shared/mode/taiko.png) Mocha
 
-#### Genere
+*Per il post nelle notizie, consulta: [La nuova mascotte di osu!taiko é qui!](https://osu.ppy.sh/home/news/2017-05-25-the-new-osutaiko-mascot-is-here)*
 
-Femmina
+![Mocha](img/Mocha.png "Mocha")
 
-#### Data di inizio
+Mocha é la mascotte di osu!taiko. Inizialmente é stata disegnata durante il [sesto concorso di fantart](https://osu.ppy.sh/community/contests/2) da [Crowie](https://osu.ppy.sh/users/6894067), che si é classificato 21esimo.
```
Comment (Sclafus) [inline wiki/Mascots/it.md:39]:
```suggestion
Mocha è la mascotte di osu!taiko. Inizialmente è stata disegnata durante il [sesto concorso di fantart](https://osu.ppy.sh/community/contests/2) da [Crowie](https://osu.ppy.sh/users/6894067), che si è classificato 21esimo.
```

File: wiki/Mascots/it.md [hunk 5]
```diff
@@ -1,290 +1,101 @@
...
-#### Peso
-
-?
-
-#### Dati anagrafici
-
-[Introduzione](https://osu.ppy.sh/home/news/2016-04-20-meet-maria-osumanias-new-mascot)
-
-## Cameos
+## Cameo
```
Comment (Sclafus) [inline wiki/Mascots/it.md:41]:
```suggestion
## Camei
```
Not really sure about it

File: wiki/Mascots/it.md [hunk 6]
```diff
@@ -1,290 +1,101 @@
...
-Maschio
-
-#### Data di inizio
-
-2008 \[Ritirato:2014\]
-
-#### Peso
-
-?
+>Leader appassionato della squadra delle cheerleader. Ha un'anima generosa, aiuta chi é in difficoltá confortandoli e lottando per loro.
```
Comment (Sclafus) [inline wiki/Mascots/it.md:47]:
```suggestion
>Leader della squadra delle cheerleader. Ha un'anima generosa, aiuta chi è in difficoltà confortandoli e lottando per loro.
```

File: wiki/Mascots/it.md [hunk 7]
```diff
@@ -1,290 +1,101 @@
...
 
-#### Dati anagrafici
+一本木龍太 (Ryūta Ippongi) ha raccolto frutta in osu!catch dal 2008 fino al 2014, dove poi é stato sostituito da [Yuzu](#yuzu). É stato creato dalla [iNiS Corporation](https://en.wikipedia.org/wiki/INiS) e faceva parte del vecchio sito web. Compare anche in [Yandere Simulator](https://yanderesimulator.com) come un NPC.
```
Comment (Sclafus) [inline wiki/Mascots/it.md:49]:
```suggestion
一本木龍太 (Ryūta Ippongi) ha raccolto frutta in osu!catch dal 2008 fino al 2014, dove poi è stato sostituito da [Yuzu](#yuzu). È stato creato dalla [iNiS Corporation](https://en.wikipedia.org/wiki/INiS) e faceva parte del vecchio sito web. Compare anche in [Yandere Simulator](https://yanderesimulator.com) come un NPC.
```

File: wiki/Mascots/it.md [hunk 8]
```diff
@@ -1,290 +1,101 @@
...
+### Agente J
 
-![Agent J](img/Agent_J.png "Agent J")
+![Agente J](img/Agent_J.png "Agente J")
 
-#### Nome reale
+> Esperto in molto stili di danza che vanno dall'hip-hop al balletto, J puó incantare qualsiasi essere vivente.
 
-Agent J
+Agente J, conosciuto anche come BA-2 (Beat Agent-2) o J, é stata una delle mascotte di osu! dal 2008, per poi ritirarsi nel 2014, E'stato creato dalla iNiS Corporation e faceva parte del vecchio sito web.
```
Comment (Sclafus) [inline wiki/Mascots/it.md:59]:
```suggestion
Agente J, conosciuto anche come BA-2 (Beat Agent-2) o J, è stata una delle mascotte di osu! dal 2008 fino al 2014. È stato creato dalla iNiS Corporation e faceva parte del vecchio sito web.
```

File: wiki/Mascots/it.md [hunk 9]
```diff
@@ -1,290 +1,101 @@
...
-和田どん (Wada Don)
-
-#### Alias
-
-Don, Don-chan
-
-#### Compleanno
-
-?
+> Don é il protagonista della serie [Taiko no Tatsujin](https://en.wikipedia.org/wiki/Taiko_no_Tatsujin). Ë un tamburo taiko con il bordo bianco e quattro gambe, un viso rosso (che sembra di un arancione acceso), e un corpo blu chiaro. Il sogno di Don é condividere la bellezza di Taiko con il resto del mondo. Sono passati 3 anni da quando si é trasferito nelle Wada House, e é diventato abbastanza popolare nella cittadina. Ha un appetito mostruoso, e alcune volte fa spese costose nella Wada House, il che può portare al peggio. Molto spesso finisce le sue frasi con "Ta-don", che significa "Ba-dum" in Giapponese.
```
Comment (Sclafus) [inline wiki/Mascots/it.md:67]:
```suggestion
> Don è il protagonista della serie [Taiko no Tatsujin](https://en.wikipedia.org/wiki/Taiko_no_Tatsujin). È un tamburo taiko con il bordo bianco e quattro gambe, un viso rosso (che sembra di un arancione acceso), e un corpo blu chiaro. Il sogno di Don è condividere la bellezza di Taiko con il resto del mondo. Sono passati 3 anni da quando si è trasferito nelle Wada House, ed è diventato abbastanza popolare nella cittadina. Ha un appetito mostruoso, e alcune volte fa spese costose nella Wada House, il che può portare al peggio. Molto spesso finisce le sue frasi con "Ta-don", che significa "Ba-dum" in Giapponese.
```

File: wiki/Mascots/it.md [hunk 10]
```diff
@@ -1,290 +1,101 @@
...
 
-#### Altezza
+和田どん (Wada Don), conosciuto anche come Don o Don-chan, é stata una della mascotte di osu!taiko da Maggio 2008. É alto 48cm e pesa piú di 100kg. Compare nella skin di osu!taiko. Il suo disegno é stato fatto da Yukiko Yokoo (横尾有希子) e gli é stata data una voce da Narahashi Miki (楢橋 美紀).
```
Comment (Sclafus) [inline wiki/Mascots/it.md:69]:
```suggestion
和田どん (Wada Don), conosciuto anche come Don o Don-chan, è stata una della mascotte di osu!taiko da Maggio 2008. È alto 48cm e pesa più di 100kg. Compare nella skin di osu!taiko. Il suo disegno è stato fatto da Yukiko Yokoo (横尾有希子) e gli è stata data una voce da Narahashi Miki (楢橋 美紀).
```

File: wiki/Mascots/it.md [hunk 11]
```diff
@@ -1,290 +1,101 @@
...
+## Community
 
-#### Aspetto
+### Aiko
 
-- Taiko (Skin)
+![Aiko](img/Aiko.png "Aiko")
 
-{Skin ufficiale per [Options's trigger](/wiki/Client/Options#Skin)}
+Disegnata da [JMC](https://osu.ppy.sh/users/774010), Aiko era una delle partecipanti del design contest delle mascotte di osu!taiko. É una ragazza energica con una passione per osu!taiko, anche se non é la migliore! Indossando un paio di "Tabi" eleganti e una serie di accessori di Pippidon, la mascotte del passato vive in questa ragazza piena d'energia.É piuttosto bassa, con soli 154cm di altezza, ed é nata il 6 aprile 1999.
```
Comment (Sclafus) [inline wiki/Mascots/it.md:77]:
```suggestion
Disegnata da [JMC](https://osu.ppy.sh/users/774010), Aiko era una delle partecipanti del design contest delle mascotte di osu!taiko. È una ragazza energica con una passione per osu!taiko, anche se non è la migliore! Indossando un paio di "Tabi" eleganti e una serie di accessori di Pippidon, la mascotte del passato vive in questa ragazza piena d'energia. È piuttosto bassa, con soli 154cm di altezza, ed è nata il 6 aprile 1999.
```

File: wiki/Mascots/it.md [hunk 12]
```diff
@@ -1,290 +1,101 @@
...
 
-#### Creator
+### Alisa
 
-- Yukiko Yokoo (横尾有希子) \[Art\]
-- Narahashi Miki (楢橋 美紀) \[Voce\]
+![Alisa](img/Alisa.png "Alisa")
 
-#### Genere
+Disegnata da [\[ Glitch \]](https://osu.ppy.sh/users/3781400), Alisa era una delle partecipanti del design contest delle mascotte di osu!taiko. Gioca ad osu!taiko fin da bambina. Adora suonare canzoni per gli altri, ama la musica, e quando non sta dormendo o mangiando, gioca ad osu!taiko oppure a qualche videogioco retró per divertimento!
```
Comment (Sclafus) [inline wiki/Mascots/it.md:83]:
```suggestion
Disegnata da [\[ Glitch \]](https://osu.ppy.sh/users/3781400), Alisa era una delle partecipanti del design contest delle mascotte di osu!taiko. Gioca ad osu!taiko fin da bambina. Adora suonare melodie per gli altri e ama la musica. Quando non sta dormendo o mangiando, gioca ad osu!taiko oppure a qualche videogioco retró per divertimento!
```

**Review by Sclafus — CHANGES_REQUESTED** [review]

**Review by Sclafus — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**